### PR TITLE
Feature/deploy buttons

### PR DESCRIPTION
**Pull request by lukasz-bien-rmtcfm-com**

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -1,29 +1,12 @@
-# client
+# Redis shopping cart UI
```
Comment (rafal-ksiazek-rmtcfm-com) [inline client/README.md:1]: For sure not shopping cart

File: server/.env.example [hunk 2]
```diff
@@ -1,4 +1,4 @@
-SERVER_PORT=3000
+PORT=3000
 
 REDIS_HOST=127.0.0.1
 REDIS_PORT=6379
```
Comment (rafal-ksiazek-rmtcfm-com) [inline server/.env.example:4]: I don't see REDIS_PASSWORD here

File: client/README.md (hunk 1 above)
Comment (lukasz-bien-rmtcfm-com) [reply client/README.md]: done

File: server/.env.example (hunk 2 above)
Comment (lukasz-bien-rmtcfm-com) [reply server/.env.example:4]: done

**Review by rafal-ksiazek-rmtcfm-com — APPROVED** [review]

**Pull request merged**